package.json

```diff
@@ -537,14 +541,19 @@
           "when": "view == mongoDBConnectionExplorer && viewItem == connectedConnectionTreeItem",
           "group": "3@2"
         },
+        {
+          "command": "mdb.openWorkspaceSettingsFile",
```

Comment:
I think it'd be nice to include ability to quickly open workspace settings here (piggybacking on an existing VSCode action). 
We can't easily detect where this preset comes from (i.e. it could also come from user settings) so we could also include a shortcut to open user settings, but maybe we don't want to encourage that and assume they know what they're doing by that point?